build.gradle

```diff
@@ -243,13 +241,16 @@ task pythonFormatterPreCommit() {
   dependsOn 'sdks:python:test-suites:tox:py38:formatter'
 }
 
+task pythonPostCommit() {
```

Comment:
This task `pythonPostCommit` aims to trigger tasks whose python versions defined in gradle.properties, which should not to be triggered by PostCommit tasks with versions like `python2PostCommit`.
We must reconsider this task configuration when introducing high/low priority.

Did I answer your comment?

Reply:
(missed this comment) I think I understand what you mean.
Let's consider a contributor who is adding a new test or test suite that needs to run in postcommit (after PR is merged). How would they configure their suite? Looking at this file it may not be obvious, for example hdfs test suite is added in PostCommit and Py38 PostCommit but MongoDBIO suite is only defined in PostCommit. 
How can we make the configuration more straightforward?
We should also consider what guidance we can give contributors when to create a new jenkins job and when to add to add to existing postcommit jobs. For example we have a separate job for MongoDB IO, but do not for HDFS, probably because both tests were added by different people and the configuration was not easy to figure out.

Reply:
I think adding a new Jenkins job in this PR might add more confusion until we clarify how to reconcile this suite with other postcommit suites. 

How about we keep the test suites in the jenkins jobs where they are defined right now, but make the python versions configurable in the gradle.properties file? This would allow us to merge most of these changes faster, including the refactoring to use suites definitions from common.gradle.

Later we can reconsider the shapes of Jenkins job when we define High Priortity/Low priority versions.

Reply:
Thank you for detail description.
It did not occur to me.